doc/index.rst

```diff
@@ -29,15 +29,22 @@ or you can bring it up and down manually:
 
 .. code-block:: bash
 
-    $ sudo service myrobot start
-    $ sudo service myrobot stop
+    $ sudo systemctl start myrobot
+    $ sudo systemctl stop myrobot
+
+You can check if the job is currently running, stopped, or if it is having
+errors by checking the status of the job:
+
+.. code-block:: bash
+
+    $ sudo systemctl status myrobot
 
 If the job is crashing on startup, or you otherwise want to see what is
-being output to the terminal on startup, check the upstart log:
+being output to the terminal on startup, check the systemd log:
 
 .. code-block:: bash
 
-    $ sudo tail /var/log/upstart/myrobot.log -n 30
+    $ sudo journalctl -n 30 -b -u myrobot
```

Comment:
It may be worthwhile to note that in Ubuntu, the journal is not persistent across boots by default.

Reply:
I can add something like this.  Good thing to note